modules/datagrid/73/launch/added/launch/infinispan-config.sh

```diff
@@ -0,0 +1,745 @@
+                  <sasl server-name=\"${sasl_server_name}\" mechanisms=\"DIGEST-MD5\" qop=\"auth\">\
+                      <policy>\
+                          <no-anonymous value=\"true\"/>\
+                      </policy>\
+                      <property name=\"com.sun.security.sasl.digest.utf8\">true</property>\
+                  </sasl>\
+              </authentication>"
+          fi
+          if [ -n "$HOTROD_ENCRYPTION" ]; then
+            if [ -n "$ENCRYPTION_REQUIRE_SSL_CLIENT_AUTH" ]; then
+              require_ssl_client_auth="require-ssl-client-auth=\"$ENCRYPTION_REQUIRE_SSL_CLIENT_AUTH\""
+            fi
+
+            encryption="\
+              <encryption security-realm=\"ApplicationRealm\" $require_ssl_client_auth/>"
+          fi
+
+          hotrod="\
+            <hotrod-connector cache-container=\"clustered\" socket-binding=\"hotrod-internal\" name=\"hotrod-internal\">$authentication $encryption\
+            </hotrod-connector>"
+          if [ -n "$topology" ]; then
+            hotrod+="\
+            <hotrod-connector cache-container=\"clustered\" socket-binding=\"hotrod-external\" name=\"hotrod-external\">$topology $authentication $encryption\
+            </hotrod-connector>"
+          fi
+        ;;
+        "memcached")
+          if [ -n "$MEMCACHED_CACHE" ]; then
+            memcached="\
+            <memcached-connector cache-container=\"clustered\" cache=\"${MEMCACHED_CACHE}\" socket-binding=\"memcached\"/>"
+          else
+            log_warning "The cache for memcached-connector is not set so the connector will not be configured."
+          fi
+        ;;
+        "rest")
+          rest_security_realm="security-realm=\"ApplicationRealm\""
+
+          if [ -n "$REST_SECURITY_DOMAIN" ]; then
+            rest_security_realm="security-realm=\"$REST_SECURITY_DOMAIN\""
+            rest_authentication="<authentication $rest_security_realm auth-method=\"BASIC\"/>"
+          fi
+
+          if [ -n "${HTTPS_NAME}" -a -n "${HTTPS_PASSWORD}" -a -n "${HTTPS_KEYSTORE_DIR}" -a -n "${HTTPS_KEYSTORE}" ] ; then
+            if [ -n "$REST_SECURITY_DOMAIN" ]; then
+              encryption="<encryption security-realm=\"$REST_SECURITY_DOMAIN\" />"      
+            else
+              encryption="<encryption security-realm=\"ApplicationRealm\" />"
+            fi
+
+            rest="\
+                <rest-connector name=\"rest-ssl\" socket-binding=\"rest-ssl\" cache-container=\"clustered\"> \
+                   $rest_authentication \
+                   $encryption \
+                </rest-connector>"
+          fi
+
+          rest="$rest \
+            <rest-connector name=\"rest\" socket-binding=\"rest\" cache-container=\"clustered\"> \
+               $rest_authentication \
+            </rest-connector>"
+        ;;
+      esac
+    done
+  fi
+
+  subsystem="\
+        <subsystem xmlns=\"urn:infinispan:server:endpoint:8.1\">$hotrod $memcached $rest\
```

Comment:
This will have to be updated to reference 9.4 once the `clustered-openshift.xml` has been updated to use the latest version.